dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunt.mo

```diff
@@ -0,0 +1,43 @@
+within Dynawo.Electrical;
```

Comment:
From what I understand the shunt model is "alone" in the Electrical package. @adriguir is it what you had in mind?
I propose that we create a sub-package named "Shunts" in Electrical and inside we can have a ShuntQ (this model) and a ShuntB ( a fixed susceptance model) if we need it one day. But this is just a suggestion!

Reply:
Sorry, my fault ;(
I was not clear enough in my answer to @MarNusch. Indeed, @MarianneSaugierRTE you are right, I imagined a package Shunt. This would be great if you can modify this.

Reply:
Ok, I created a new sub-package "Shunts". The shunt has a fixed susceptance but a reactive power setpoint and not a setpoint for the susceptance. Shall the name of this shunt model be ShuntQfromU or something else? I think ShuntQ might give the impression of a fixed Q.

Reply:
No preference on my side for the name, I'm not very good at this!

Reply:
@MarianneSaugierRTE: Do you have any preference?

Reply:
You're right @MarNusch the name ShuntQ could be misleading.

If I had done it with the Dynawo spirit, I would have created one dynamic model ShuntB with a BPu parameter, and two init models: one classical, and one calculating the BPu from a QRef input for example. Because at the end, you are right that this is a fixed susceptance model.

Considering the other comment of @adriguir suggesting to compute Q0 at U0 from QRef at UNom, I can propose to go one step further and to compute BPu from the QRef, so that we could have our shunt model called ShuntB that could stay very explicit and generic. Would it be ok for you? You could for example extend this ShuntB to redefine a ShuntInitQRef for your specific use, inside your test case, a bit like @adriguir showed us yesterday for the loads or lines with extra switchOff equations...

What do you think @MarNusch and @adriguir ? If you don't like this idea, you can stick to what we agreed on and choose the name you prefer, as I don't have a clear preference...

Reply:
You are right.
At the end, what is a shunt model is not what I previously wrote: Q = Q0 * (U/U0)². The most generic implementation and the one that is natural is Q = BU². 

So I will keep one model just called ShuntB (as proposed by @MarianneSaugierRTE ) and in the model implementation, I would only have a parameter B (and no longer Q0 and U0 not needed anymore) and the equation Q = BU².
And instead of calculating Q0 and U0 from QRef and URef as I suggested in a previous comment, you will calculate B from QRef and URef in the test case when you pass parameters to the model.For the Modelica part, this will be what we need.

@MarianneSaugierRTE as you said, for Dynawo use, we will add later on one initialization method to calculate B from U0 and Q0 if necessary.

Reply:
I renamed the Shunt model into ShuntB, deleted the parameters Q0Pu and U0Pu, added the parameter BPu and changed the calculation from QPu = Q0Pu*(UPu/U0Pu)^2 to QPu = BPu*UPu.